lib/interface_quda.cpp

```diff
@@ -5073,6 +5092,179 @@ void computeCloverForceQuda(void *h_mom, double dt, void **h_x, void **, double
   profileCloverForce.TPSTOP(QUDA_PROFILE_TOTAL);
 }
 
+void computeTMCloverForceQuda(void *h_mom, void **h_x, double *coeff, int nvector, 
+    void *h_gauge, QudaGaugeParam *gauge_param, QudaInvertParam *inv_param)
```

Comment:
@Marcogarofalo  This `h_gauge` variable here is causing an unused variable warning which results in the build tests failing (CI builds use `STRICT` building which promotes all warnings to errors).  To fix this, either:
* delete the variable from the argument list
* just write it as `void *` instead of `void *h_gauge`, e.g., leave it unnamed
I suspect we can rely on having a resident gauge field, so you could just delete the variable from the argument list.

Reply:
(To see the compiler warnings, click on a failing build (the red cross) and you should be able to see the console build output after a few other clicks.  From there you can see the build failures, to allow you to fix any failures.)

Reply:
Thank you for the explanation. I didn't fix it immediately because I got distracted by the fact that the tuner was not working after merging with develop
```
ERROR: Failed to find key entry (N4quda11CloverForceIdLi3EL21QudaReconstructType_s18EEE:4x4x4x4:GPU-offline,vol=256stride=128precision=8geometry=4Nc=3,exterior) (rank 1, host lnode17.cluster.hiskp, tune.cpp:1065 in quda::TuneParam quda::tuneLaunch(quda::Tunable&, QudaTune, QudaVerbosity)())
       last kernel called was (name=N4quda11CloverForceIdLi3EL21QudaReconstructType_s18EEE,volume=4x4x4x4,aux=GPU-offline,vol=256stride=128precision=8geometry=4Nc=3,exterior)
```

Reply:
The tuner issue went away by making a new build directory. Still `computeTMCloverForceQuda` is not working in single precision and the test  I think is not compiling if `MULTI_GPU` is not enabled

Reply:
@maddyscientist I tried to fix the errors in the integration tests but I think the last one is not under my control.

Reply:
Ok, mind if I take a pass at cleaning up these issues then?

Reply:
Please go ahead, thanks.